index.html

```diff
@@ -19,22 +19,30 @@
         <link
             href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@500;700&display=swap"
             rel="stylesheet">
+            <script src="script.js"></script>
 </head>
 
 <body>
 <main>
     <!----------navbar setion------------------->
     <header>
     <nav class="nav-container">
-        <a class="Johnd" href=""></a>
-        <a class="menu" href="#"></a>
-        <ul>
-            <li><a href="">Portfolio</a></li>
-            <li><a href="">About</a></li>
-            <li><a href="">Contact</a></li>
-            <li><a href=""><img src="/images/msg.png"></a></li>
+       
+        <a class="Johnd" href="#"></a>
+        <a class="menu" href="#" onclick="displayPopUp()"></a>
+        <ul class="menu-items">
+            <li><a href="#my-works">Portfolio</a></li>
+            <li><a href="#about-me">About</a></li>
+            <li><a href="#contact-me">Contact</a></li>
+            <li><a href="#"><img src="./images/msg.png"></a></li>
```

Comment:
- [ ] as the previous reviewer asked you mobile menu should cover the whole window in every screen mobile so that it be responsive so please make sure to do that 
<img width="342" alt="Screenshot 2023-02-06 at 15 34 25" src="https://user-images.githubusercontent.com/61790587/217000666-80ab63ee-a20e-4ac4-9b5d-32f19be1bcdf.png">